.gitattributes

```diff
@@ -14,6 +14,7 @@
 /Tests/ export-ignore
 /eslint.config.json export-ignore
 /package.json export-ignore
+/package-lock.json export-ignore
```

Comment:
We also need to add this file to the list of files in the `prepare-release` Composer script (to keep it out of the TER when someone does a manual release).

Reply:
ack. will do so.

Reply:
done.